cve_bin_tool/config_generator.py

```diff
@@ -0,0 +1,132 @@
+class config_generator:
+    def config_generate_yaml():
+        yaml_str = """
+            # This is an automatically generated configuration file for the CVE Binary Tool. It allows you to customize and manage the tool to suit your needs.
+            # Please exercise caution when editing this file and follow the instructions provided.
+            # To generate a new configuration file, use the --generate-config option. For more information, please refer to the official CVE Binary Tool documentation at https://cve-bin-tool.readthedocs.io/en/latest/.
+            # This file enables you to specify options such as the installation directory of the tool, the data sources to be used, and other relevant settings. To make changes, simply modify the values to the right of the colon sign.
+            # For more information on the available options and how to configure them, please refer to the official documentation at https://cve-bin-tool.readthedocs.io/en/latest/.
+            # If you support the project and wish to contribute, you can find the official CVE Binary Tool Contributor Guide at https://cve-bin-tool.readthedocs.io/en/latest/CONTRIBUTING.html#cve-binary-tool-contributor-guide.
+            # And link for project github https://github.com/intel/cve-bin-tool
+
+            cve_data_download:
+              #set your nvd api key
+              nvd_api_key:your-key-here
+
+            input:
+              # Directory to scan
+              directory: test/assets
+              # To supplement triage data of previous scan or run standalone as csv2cve
+              # Currently we only support csv and json file.
+              input_file: test/csv/triage.csv
+
+            checker:
+              # list of checkers you want to skip
+              skips:
```

Comment:
Ditto for skips and runs values: these would be surprising defaults; we probably want to have it commented out to show the syntax without changing behaviour.